QualityControl/public/common/filters/model/FilterModel.js

```diff
@@ -183,9 +182,7 @@ export default class FilterModel extends Observable {
    * @returns {Promise<void>}
    */
   async deactivateRunsMode(baseViewModel) {
-    this._filterMap = this._previousFilterMap || {};
     this.resetRunsMode();
-    this.setFilterToURL();
     await baseViewModel.triggerFilter();
```

Comment:
Given that now you are not restoring the previous filter anymore. For example if the user has runTypes set and then enters runMode, the run types will not be set anymore. 
Thus, I do not see the point of calling `triggerFilter` again as the objects/layout displayed will be the same as in RunMode. What do you think?